ansible/group_vars/omeroreadonly-hosts.yml

```diff
@@ -11,3 +11,13 @@ idr_omero_web_user_dropdown: false
 # Set this to 'ro' to mount /data/BioFormatsCache on read-only servers as
 # read-only, default is 'rw'
 # idr_bioformatscache_readwrite:
+
+# MS variables
+
+omero_ms_image_region_update_nginx: true
+omero_ms_image_region_db_url:  {{ omero_db_host_ansible }}
```

Comment:
See https://github.com/IDR/deployment/actions/runs/10549799844/job/29225266076

```
  Syntax Error while loading YAML.
    found unacceptable key (unhashable type: 'AnsibleMapping')
  
  The error appears to be in '/home/runner/work/deployment/deployment/ansible/group_vars/omeroreadonly-hosts.yml': line 18, column 33, but may
  be elsewhere in the file depending on the exact syntax problem.
  
  The offending line appears to be:
  
  omero_ms_image_region_update_nginx: true
  omero_ms_image_region_db_url:  {{ omero_db_host_ansible }}
                                  ^ here
  We could be wrong, but this one looks like it might be an issue with
  missing quotes. Always quote template expression brackets when they
  start a value. For instance:
  
      with_items:
        - {{ foo }}
  
  Should be written as:
  
      with_items:
        - "{{ foo }}"
```